src/TorchSharp/TorchVision/IO/Image.cs

```diff
@@ -1,3 +1,4 @@
+// Copyright (c) .NET Foundation and Contributors.  All Rights Reserved.  See LICENSE in the project root for license information.
```

Comment:
Some may want equivalents of `pil_to_tensor()` and `tensor_to_pil()` to convert to/from `SkImage` or `ImageSharp.Image` directly. Not necessarily included in this PR.

https://pytorch.org/vision/stable/generated/torchvision.transforms.functional.pil_to_tensor.html#torchvision.transforms.functional.pil_to_tensor

Reply:
I really would prefer not to allow types from underlying image libraries to escape into the TorchSharp public APIs -- that creates a tight coupling that will be hard to walk away from later. 

If converting to images (Skia, ImageSharp, or others) is important, I would prefer to provide well-documented example code that shows how to do it. There's already an example for ImageSharp, and the SkiaSharp code was lifted from the example I had in place.

Reply:
I thought you wanted to make SkiaSharp the first-class library by adding SkiaSharp dependency to TorchSharp(Vision). If TorchSharp API should be clean from SkiaSharp. I think the package should be divided into TorchSharp and TorchSharp.Skia instead.

Reply:
I think that's Python way. They make all things included so that novice programmers can do many things out of box. I think C# libraries tend to have less dependencies, that enables wider platform support. That's one of its strong points.